weblab_cg/_generator_chaste_cg.py

```diff
@@ -138,6 +143,20 @@ def __init__(self, model, class_name, file_name):
         self._formatted_y_derivatives = self._format_y_derivatives()
         self._formatted_derivative_eqs = self._format_derivative_equations()
         self._free_variable, self._ode_system_information, self._named_attributes = self._format_ode_system_info()
+        self.expose_annotated_variables = False  # Using property to initialise and format derived quantity equations
+
+    @property
+    def expose_annotated_variables(self):
+        return self._expose_annotated_variables
+
+    @expose_annotated_variables.setter
+    def expose_annotated_variables(self, value):
+        self._expose_annotated_variables = value
+        self._derived_quant = self._get_derived_quant()
+        self._formatted_derived_quant = self._format_derived_quant()
+        self._derived_qunat_eqs = self._get_derived_qunat_eqs()
```

Comment:
Check your spelling!

Reply:
looks good to me! (quant is a deliberate shortening)

Reply:
qunat != quant :)

Reply:
o now I see it